Source/Demo/ArithmeticFloatExtractor.h

```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+#include "CoreMinimal.h"
+#include "FloatExtractor.h"
+#include "ArithmeticFloatExtractor.generated.h"
+
+/**
+ * FloatExtractor that changes value by some delta every time FloatExtractor is called
+ */
+UCLASS()
+class DEMO_API UArithmeticFloatExtractor : public UFloatExtractor
+{
+	GENERATED_BODY()
+	
+public:
+	virtual float GetValue_Implementation() override;
+
+	UPROPERTY(EditAnywhere, BlueprintReadWrite)
```

Comment:
Specify categories for all properties and functions in the plugin. Use separations. For example: Category="QoL|Float Extractor" or "QoL|Float Extractor|Arithmetic"